server/src/models/gitlab.rs

```diff
@@ -26,6 +26,17 @@ impl PipelineUpdate {
             return None;
         }
 
+        if !CONFIG.target_variable.is_empty()
+            && !self
+                .object_attributes
+                .variables
+                .iter()
+                .any(|e| e.key == CONFIG.target_variable)
+        {
+            tracing::debug!("Variable was not found '{}'", CONFIG.target_variable);
```

Comment:
Be explicit what was the expected variable to be found like https://github.com/Songtronix/Airshipper/blob/7ba15179df49063c3fb6b3976b061e7dcd3516e3/server/src/models/gitlab.rs#L22

Reply:
i am already listing the expected variable to be found. do you want to to list the vec of all provided variables, even if it can be very large?